src/main/java/com/example/icecreamup/dto/IngredientDTO.java

```diff
@@ -10,7 +10,7 @@
 	private int ingredientId;
 	private String ingredientName;
 	private boolean isAvailable;
-	private double price;
+	private double ingredientPrice;
```

Comment:
Changed naming convention of this variable the related methods, as price methods have been added to other classes.